specification/common-types/resource-management/v6/privatelinks.json

```diff
@@ -106,7 +129,24 @@
       ],
       "x-ms-enum": {
         "name": "PrivateEndpointServiceConnectionStatus",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
```

Comment:
I believe "Disconnected" is also supposed to be an allowed value.

Reply:
Do we need to add v7 to document this? :-/

Reply:
I think this would be moved as a regular common-types update, I don't know which is the policy for these changes

Reply:
in the ARM/TypeSpec/SDK CodeGen bi-weekly sync meeting we discussed this issue and this is not going to be part of the migration effort, but this can be added as part of v6 as is not breaking change if needed.